src/app/pages/test-report/cat-b/components/manoeuvres-popover/manoeuvres.popover.scss

```diff
@@ -0,0 +1,34 @@
+.gds-radio-button {
```

Comment:
previous files was wrapped in `manoeuvres-popover{}` need to keep this but swap for `:host{}` (see conf page)